sympy/assumptions/satask.py

```diff
@@ -29,6 +29,15 @@ def satask(proposition, assumptions=True, context=global_assumptions,
         return False
 
     if not can_be_true and not can_be_false:
+        from pprint import pprint
+        pprint(can_be_true)
+        pprint(can_be_false)
+        pprint('proposition')
+        pprint(proposition)
+        pprint('assumptions')
+        pprint(assumptions)
+        pprint('relevant_facts')
+        pprint(relevant_facts)
```

Comment:
It's best to avoid committing debug code if possible. This doesn't fail the tests, and it's quite easy to accidentally merge it in. I always use `git commit -i` to do partial commits so that I can leave things like this out.